browserslist

```diff
@@ -0,0 +1,6 @@
+last 2 Chrome versions
+last 2 Firefox versions
+last 2 Safari versions
+last 2 Edge versions
+IE 11
+>1%
```

Comment:
Do we know what `>1%` globally translates too? I think we should get rid of this if not, because it seems like something that can change without us formally committing to it?

Reply:
You can preview `>1%` (or any other `browserslist` config) [here](http://browserl.ist/?q=%3E1%25). Without it, I would have to add verbose entries for browsers like iOS Safari. I can move in that direction though if you'd prefer it to be more explicit.